src/overlays/actors/ovl_Bg_Gnd_Darkmeiro/z_bg_gnd_darkmeiro.h

```diff
@@ -10,7 +10,10 @@ typedef void (*BgGndDarkmeiroActionFunc)(struct BgGndDarkmeiro*, GlobalContext*)
 
 typedef struct BgGndDarkmeiro {
     /* 0x0000 */ DynaPolyActor dyna;
-    /* 0x0164 */ char unk_164[0x8];
+    /* 0x0164 */ u16 actorState; // Bitflags for whether a platform or timer is on. 0x8 for timer 2, 0x4 for timer 1, 0x2 for whether a platform is on.
+	/* 0x0166 */ s16 timer1; //Timer for switch flag N+1. Platforms use it to count down to vanishing after the switch flag is unset.
+	/* 0x0168 */ s16 timer2; //Timer for switch flag N+2. Platforms use it to control their transparency (does not work).
```

Comment:
Since the format.sh script won't format .h files, you'll have to make sure you ident the headers right manually.
```suggestion
    /* 0x0164 */ u16 actorState; // Bitflags for whether a platform or timer is on. 0x8 for timer 2, 0x4 for timer 1, 0x2 for whether a platform is on.
    /* 0x0166 */ s16 timer1; //Timer for switch flag N+1. Platforms use it to count down to vanishing after the switch flag is unset.
    /* 0x0168 */ s16 timer2; //Timer for switch flag N+2. Platforms use it to control their transparency (does not work).
```